WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -218,6 +220,26 @@ class AbstractPostListViewController: UIViewController,
         return properties
     }
 
+    // MARK: - Notifications
+
+    @objc private func postCoordinatorDidUpdate(_ notification: Foundation.Notification) {
+        guard let updatedObjects = (notification.userInfo?[NSUpdatedObjectsKey] as? Set<NSManagedObject>) else {
+            return assertionFailure("missing NSUpdatedObjectsKey")
+        }
+        let updatedIndexPaths = (tableView.indexPathsForVisibleRows ?? []).filter {
+            let cell = tableView.cellForRow(at: $0) as? AbstractPostListCell
```

Comment:
Changes compared to the previous implementation:
- Will no longer crash when `postCoordinatorDidUpdate` calls are not synchronized wit the reload of the list (rare exciting `fetchResultsController.object(at: $0)` crash)
- Will work for pages where the cells are not displayed in the same order as they are present in fetchResultsController